services/authoring/src/utils/jsonLogger.ts

```diff
@@ -0,0 +1,55 @@
+import morgan = require('morgan');
+import express = require('express');
+
+const getLevelByStatus = (status: string): string => {
+    let result: string;
+    const intStatus = parseInt(status, 10);
+    switch (intStatus / 100) {
+        case 4:
+            result = 'Warning';
+            break;
+        case 5:
+            result = 'Error';
+            break;
+        case 1:
+        case 2:
+        case 3:
+        default:
+            result = 'Info';
+            break;
+    }
+    return result;
+};
+
+const levels: Array<any> = [
+    { method: 'info', level: 'Info' },
+    { method: 'log', level: 'Info' },
+    { method: 'warn', level: 'Warning' },
+    { method: 'error', level: 'Error' },
+    { method: 'trace', level: 'Trace' },
+];
+
+export const logger = levels.reduce((acc, log) => ({
+    ...acc,
+    [log.method || log]: (Message, ExtraData) => console.log(JSON.stringify({ Level: log.level || log, Message, ExtraData }))
+  }), {});
+
+export const morganJSON = morgan((tokens, req, res) => {
```

Comment:
Separate morgan from function and then add test for function.

Reply:
We decided to drop the testing of this function